packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/ChipGroupDemo/ChipGroupDemo.tsx

```diff
@@ -46,7 +46,7 @@ export class BadgeChip extends Component<{}, BadgeChipState> {
   render() {
     const { badgeChipArray } = this.state;
     return (
-      <ChipGroup>
+      <ChipGroup defaultIsOpen={false}>
```

Comment:
Can you add a demo for the defaultIsOpen being true and a test for that as well.